src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -1361,10 +1361,9 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
         }
         case SMART_ACTION_SET_DISABLE_GRAVITY:
         {
-            if (!IsSmart())
-                break;
-
-            ENSURE_AI(SmartAI, me->AI())->SetDisableGravity(e.action.setDisableGravity.disable != 0);
+            for (WorldObject* target : targets)
```

Comment:
Changes behavior

Does this not require sql updates to make existing actions target the same object?

Reply:
All action targets were already 1 (except for one). Added sql for that case (target type 0) and custom cases